Firestore/core/src/firebase/firestore/util/filesystem.h

```diff
@@ -17,6 +17,7 @@
 #ifndef FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_UTIL_FILESYSTEM_H_
 #define FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_UTIL_FILESYSTEM_H_
 
+#include <dirent.h>
```

Comment:
filesystem.h is portable code and can't include this POSIX-only header.

Reply:
Removed.